portfolio/src/main/webapp/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <title>Fiana Vicker's Professional Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="script1.js"></script>
```

Comment:
Any reason for creating a new script file rather than putting the new function in the existing one? It's not a problem, I'm just wondering.